relay-server/src/services/buffer/envelope_buffer/mod.rs

```diff
@@ -480,7 +302,10 @@ where
     /// lifecycle
     async fn load_store_total_count(&mut self) {
         let total_count = timeout(Duration::from_secs(1), async {
-            self.stack_provider.store_total_count().await
+            match &self.envelope_repository {
+                EnvelopeRepository::Memory(_) => 0,
```

Comment:
nit: The `0` is only correct if we call this function in initialization. Should we count the number of elements in the memory buffer, just for the sake of correctness?

Reply:
Yep, this is a good point.